lib/node_modules/@stdlib/math/base/special/expm1rel/test/test.js

```diff
@@ -21,15 +21,151 @@
+var smallPositive = require( './fixtures/python/small_positive.json' );
+var tiny = require( './fixtures/python/tiny.json' );
+
+
 // TESTS //
 
+tape( 'the function accurately computes `10**x` for negative medium numbers', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var v;
+	var i;
+
+	x = mediumNegative.x;
+	expected = mediumNegative.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = expm1rel( x[ i ] );
+		delta = abs( v - expected[ i ] );
+		tol = EPS * abs( expected[ i ] );
+		t.ok( delta <= tol, 'within tolerance. x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '. Delta: ' + delta + '. Tolerance: ' + tol + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function accurately computes `10**x` for positive medium numbers', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var v;
+	var i;
+
+	x = mediumPositive.x;
+	expected = mediumPositive.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = expm1rel( x[ i ] );
+		delta = abs( v - expected[ i ] );
+		tol = EPS * abs( expected[ i ] );
+		t.ok( delta <= tol, 'within tolerance. x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '. Delta: ' + delta + '. Tolerance: ' + tol + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function accurately computes `10**x` for negative small numbers', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var v;
+	var i;
+
+	x = smallNegative.x;
+	expected = smallNegative.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = expm1rel( x[ i ] );
+		delta = abs( v - expected[ i ] );
+		tol = EPS * abs( expected[ i ] );
+		t.ok( delta <= tol, 'within tolerance. x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '. Delta: ' + delta + '. Tolerance: ' + tol + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function accurately computes `10**x` for positive small numbers', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var v;
+	var i;
+
+	x = smallPositive.x;
+	expected = smallPositive.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = expm1rel( x[ i ] );
+		delta = abs( v - expected[ i ] );
+		tol = EPS * abs( expected[ i ] );
+		t.ok( delta <= tol, 'within tolerance. x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '. Delta: ' + delta + '. Tolerance: ' + tol + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function accurately computes `10**x` for very small `x`', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var v;
+	var i;
+
+	x = tiny.x;
+	expected = tiny.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = expm1rel( x[ i ] );
+		delta = abs( v - expected[ i ] );
+		tol = EPS * abs( expected[ i ] );
+		t.ok( delta <= tol, 'within tolerance. x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '. Delta: ' + delta + '. Tolerance: ' + tol + '.' );
+	}
+	t.end();
+});
+
 tape( 'main export is a function', function test( t ) {
```

Comment:
Normally, we make this test the very first test in the file. The reason being that L145 prints the file name and provides a basic test for expected export value type, meaning the test sits at the highest level of abstraction and subsequent tests typically drill down. Accordingly, all the tests currently above this test would be moved below.